File: doc/user/06_feature/gpmi/rosetta/12_hello_menu/ex.rb
PkgLoad("pcb-rnd-gpmi/actions", 0);
PkgLoad("pcb-rnd-gpmi/dialogs", 0);

def ev_action(id, name, argc, x, y)
	dialog_report("Greeting window", "Hello world!");
end

def ev_gui_init(id, argc, argv)
	create_menu("/main_menu/Plugins/GPMI scripting/hello", "hello()", "h", "Ctrl<Key>w", "tooltip for hello");
end

Bind("ACTE_action", "ev_action");
Bind("ACTE_gui_init", "ev_gui_init");
action_register("hello", "", "log hello world", "hello()");

File: doc/user/06_feature/scripting/rosetta/11_hello_menu/ex.pas
function hello();
begin
	message('hello!\n');
end;

function main(ARGV);
begin
	fgw_func_reg(hello);
	cookie := scriptcookie();
	createmenu('/main_menu/Plugins/script/hello', 'hello()', 'Prints hello! in the log', cookie);
end;

